### DISCUSSION TITLE
Adding a way to mark a combobox invalid.

### DISCUSSION THREAD

**Discussion by FoksVHox** [tailwindlabs/headlessui#3278] [Ideas]
There is currently not a way to make a Combobox invalid. Every other form element, has a `invalid`-attribute, that'll trigger the invalid styling for the element using `data-invalid`. Would be nice if this would be added to catalyst.

**Comment by aaronadamsCA** [comment]
Rather than submit a separate request, I'd like to extend this request to include checkboxes, switches and radio groups as well. All of these + comboboxes are frequently invalid if the user does not interact with them, e.g. "you must accept the terms" or "you must select a value."

Currently we're just adding a `data-invalid` attribute ourselves, which is fine, but inconsistent.
